--- a/helang/launch.py	2022-08-18 22:18:28.536530342 +0800
+++ b/helang/launch.py	2022-08-18 22:21:24.703315625 +0800
@@ -6,8 +6,8 @@ from helang.lexer import Lexer
 from helang.parser import Parser
 from helang.exceptions import HeLangException
 from helang.u8 import U8
-from helang.lt_code.window import LTCodeWindow
-from PySide6.QtWidgets import QApplication
+# from helang.lt_code.window import LTCodeWindow
+# from PySide6.QtWidgets import QApplication


 SHELL_HELP = """
@@ -58,13 +58,13 @@ def launch_shell():
             print('Fatal Error! Revise Saint He\'s videos!')
             raise e

-
+'''
 def launch_editor():
     app = QApplication()
     editor = LTCodeWindow()
     editor.show()
     sys.exit(app.exec_())
-
+'''

 def launch_great_script():
     with open('./great.he', 'r') as f:
@@ -78,7 +78,7 @@ def launch_great_script():
 LAUNCHERS = {
     'great': launch_great_script,
     'shell': launch_shell,
-    'editor': launch_editor,
+    #'editor': launch_editor,
 }
